etcdserver/raft.go

```diff
@@ -94,14 +94,7 @@ type raftNode struct {
 	term  uint64
 	lead  uint64
 
-	mu sync.Mutex
-	// last lead elected time
-	lt time.Time
```

Comment:
Copying the structure around copies the mutex which causes the static checks to complain.